index.html

```diff
@@ -179,8 +180,8 @@ <h1>Introduction</h1>
 		<li>improving the accessibility of dynamic content generated by scripts; and</li>
 		<li>providing for interoperability with <a>assistive technologies</a>.</li>
 	</ul>
-	<p>WAI-ARIA is a technical specification that provides a framework to improve the accessibility and interoperability of web content and applications. This document is primarily for developers creating custom widgets and other web application components. Please see the <a href="http://www.w3.org/WAI/intro/aria">WAI-ARIA Overview</a> for links to related documents for other audiences, such as <a href="https://www.w3.org/TR/wai-aria-practices/">WAI-ARIA Authoring Practices</a> [[WAI-ARIA-PRACTICES-1.1]] that introduces developers to the accessibility problems that WAI-ARIA is intended to solve, the fundamental concepts, and the technical approach of WAI-ARIA.</p>
-	<p>This draft currently handles two aspects of <a>roles</a>: user interface functionality and structural <a>relationships</a>. For more information and use cases, see [[WAI-ARIA-PRACTICES-1.1]] for the use of roles in making interactive content accessible.</p>
+	<p>WAI-ARIA is a technical specification that provides a framework to improve the accessibility and interoperability of web content and applications. This document is primarily for developers creating custom widgets and other web application components. Please see the <a href="http://www.w3.org/WAI/intro/aria">WAI-ARIA Overview</a> for links to related documents for other audiences, such as <a href="" class="practices">WAI-ARIA Authoring Practices</a> [[WAI-ARIA-PRACTICES-1.2]] that introduces developers to the accessibility problems that WAI-ARIA is intended to solve, the fundamental concepts, and the technical approach of WAI-ARIA.</p>
+	<p>This draft currently handles two aspects of <a>roles</a>: user interface functionality and structural <a>relationships</a>. For more information and use cases, see [[WAI-ARIA-PRACTICES-1.2]] for the use of roles in making interactive content accessible.</p>
```

Comment:
Add the nice link for consistency?
(Also changed the word "draft" to "document" because draft is quite confusing here).

```suggestion
	<p>This document currently handles two aspects of <a>roles</a>: user interface functionality and structural <a>relationships</a>. For more information and use cases, see <cite><a href="" class="practices"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices</a></cite> [[WAI-ARIA-PRACTICES-1.2]] for the use of roles in making interactive content accessible.</p>
```